src/OpenLoco/Paint/Paint.cpp

```diff
@@ -497,8 +496,8 @@ namespace OpenLoco::Paint
         }
 
         ps->imageId = imageId;
-        ps->x = vpPos.x;
-        ps->y = vpPos.y;
+        ps->vpPos.x = vpPos.x;
+        ps->vpPos.y = vpPos.y;
```

Comment:
Can this be assigned in one go, directly to vpPos?

Reply:
No. I'm still not sure what to do about the viewport_pos type. I can't decide if it should be removed and leave it to the caller to know when it needs further processing.